### PR TITLE
Clarify that that two empty cancellation tokens are always equal

### PR DESCRIPTION
**Pull request by ohadschn**
Addresses https://github.com/dotnet/corefx/issues/495#issuecomment-323999818

### CODE REVIEW

**Review by jzeferino — APPROVED** [review]
LGTM.

File: xml/System.Threading/CancellationToken.xml [hunk 1]
```diff
@@ -337,7 +337,7 @@
 ## Remarks  
  The cancellation token returned by this property cannot be canceled; that is, its <xref:System.Threading.CancellationToken.CanBeCanceled%2A> property is `false`.  
   
- You can also use the C#  [default(CancellationToken)](~/docs/csharp/language-reference/keywords/default.md) statement to create an empty cancellation token.  
+ You can also use the C#  [default(CancellationToken)](~/docs/csharp/language-reference/keywords/default.md) statement to create an empty (and equivalent) cancellation token. Two empty cancellation tokens are always equal.
```
Comment (rpetrusha) [inline xml/System.Threading/CancellationToken.xml]: Could you make the following changes:
1. Please remove "(and equivalent)" from the first sentence. It's unclear what equivalent refers to in the context of the sentence.
2. Please move the second sentence to be an independent paragraph that follows the first paragraph. The second sentence is related to the equality of CancellationTokens, not to the C# default keyword.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks, @ohadschn, for contributing to the dotnet/docs repo and for noting that two empty cancellation tokens are equal. I've requested that you make two changes to the content that you've added.

**Comment by ohadschn** [comment]
@rpetrusha thank you, I've made the requested changes.

**Comment by rpetrusha** [comment]
Thank you so much, @ohadschn, for making the additional change. We'll merge your PR now. Your changes should appear on docs.microsoft.com in the next day or two.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**